packages/core/src/client/validate.test.ts

```diff
@@ -1,7 +1,28 @@
 import { eq, onchainTable, relations } from "@/index.js";
 import { createClient } from "@ponder/client";
 import { expect, test } from "vitest";
-import { validateQuery } from "./validate.js";
+import { findTableNames, validateQuery } from "./validate.js";
+
+test("findTableNames", async () => {
```

Comment:
Would be good to see a test query that included qualified table names (with schema)